### ISSUE TITLE
order-in-components fix failed

### ISSUE THREAD

**Issue by imshenshen** [vuejs/eslint-plugin-vue#829]
**Tell us about your environment**	
* **ESLint version:**	
v5.14.1
* **eslint-plugin-vue version:**	
eslint-plugin-vue@5.2.2
* **Node version:**	
v11.7.0

**Please show your full configuration:**

```javascript
module.exports = {
  extends: ['plugin:vue/recommended', 'standard', 'prettier', 'prettier/standard'],
  plugins: ['vue', 'prettier', 'standard'],
  rules: {
    'prettier/prettier': 'error',
    'vue/no-async-in-computed-properties': 'off',
    'vue/max-attributes-per-line': 'off'
  },
  env: {
    browser: true
  },
  parserOptions: {
    parser: 'babel-eslint'
  },
  globals:{
    require:true
  }
}
```

**What did you do?**	

```shell
npx eslint client/src/components/regionRadio.vue --fix
/components/regionRadio.vue
65:3  warning  The "data" property should be above the "computed" property on line 62  vue/order-in-components

✖ 1 problem (0 errors, 1 warning)

```

**What did you expect to happen?**
[🔧 The --fix option on the command line can automatically fix some of the problems reported by this rule](https://vuejs.github.io/eslint-plugin-vue/rules/order-in-components.html)

**What actually happened?**

still warning

**Comment by ota-meshi** [comment]
Thank you for posting issue.

Could you paste your `.vue` file?

`order-in-components` does not autofix if there is a possibility of changing the behavior of the script.
Your `.vue` file may apply to this.

**Comment by homerjam** [comment]
I believe I have the same issue, here's my `.vue` file

```vue
<script>
import _ from 'lodash';

export default {
  async fetch({ store }) {
    await store.dispatch('workspaces/readMyWorkspaces');
  },
  data() {
    return {
      workspaces: _.map(this.$store.state.workspaces.myWorkspaces),
    };
  },
};
</script>

<template lang="pug">
.workspaces
  h1 Workspaces

  ul
    li(v-for="workspace in workspaces") {{ workspace.name }}
</template>
```

**Comment by imshenshen** [comment]
@homerjam Did you use [spread properties](https://github.com/tc39/proposal-object-rest-spread)? With that the fix behavior failed

**Comment by homerjam** [comment]
@imshenshen no - the code I pasted above shows the error for me

**Comment by ota-meshi** [comment]
Hello! @homerjam

Your source code problem cannot be reproduced on the [Demo].
Autofix works without problems.
Open another issue and tell us about your environment.


[Demo]: https://mysticatea.github.io/vue-eslint-demo/#eJxNUdFqwzAM/BVhBkkhSVkf065/sKcN+hIoJlZas0Q2ttOuC/n3yQlN8yKsO92dLQ+iNgpFKQ6+dtqGY0W6s8YFOEPjTAdJa5T012RfUUX4O1EKG9m3AYaKAKR/UA0NhvqaDuCDcQjjZuaYvUsdZrRQ2lsZx5K7cT98rtFvHUr1+TgtQLLZR+WYxapkkOni5TD0jp4dwMulhHPRSZuGq/bF25zmgwxYvGaKbpWyyWaTcQmLp8N2WQI3ATvbsge0ki4flbD9pRJMrTyj+PoOpxUQob6d3Vud3vLGOBYvGtC0ungl+HXDCyhIdry+Md7lmX8UmXB9i16Ug7j1uCWT14aa3mu65FMAV92IcpdNvHEKXa6Jp/grCSmwdDdmQpPi5kv/4TQ7t98PG/9/iuckK51HxwB66xDF+A9iKb7e

**Comment by homerjam** [comment]
Thanks @ota-meshi ! I'll try again to figure out the issue in my environment

**Comment by ota-meshi** [comment]
Please share the source code and reopen this issue.
